### ISSUE TITLE
Django 2.1 works not fine

### ISSUE THREAD

**Issue by LQBing** [philipn/django-rest-framework-filters#274]
I was installed it with django 2.1, and alert: 

from django.db.models.sql.constants import QUERY_TERMS

**Comment by adambrenecki** [comment]
I've run into that issue; it's solved by upgrading `django-filter` to the 2.x release series; the error isn't actually in this package.

Unfortunately, when you do so, you get a different error: 

```
E           AttributeError: 'AllLookupsFilter' object has no attribute 'name'

../../Library/Caches/pypoetry/virtualenvs/parts-py3.6/lib/python3.6/site-packages/rest_framework_filters/filterset.py:44: AttributeError
```

Upgrading to the master branch fixes this error, but it seems that the fix hasn't made it into a release yet.

**Comment by rpkilby** [comment]
Hi all, I've released v1.0.0.dev0 to PyPI. This is a developmental release, but is fairly stable.
